src/scripts/main.js

```diff
@@ -0,0 +1,120 @@
+const formContainer = document.querySelector('.form__container');
+const addButton = document.querySelector('.form__button--add');
+const submitButton = document.querySelector('.form__button--submit');
+const MAX_INPUT_COUNT = 6;
+
+const regEx = /.+?(?=[?]passwords=)/;
```

Comment:
regEx to mało mówiąca nazwa.
ba, jak piszę regexa w kodzie - to zazwyczaj dodaję komentarz co on robi.